funtofem/optimization/optimization_manager.py

```diff
@@ -24,6 +24,7 @@
 
 import os, numpy as np
 import matplotlib.pyplot as plt
+from matplotlib.ticker import MaxNLocator
```

Comment:
We could make this generalizable to be able to change the ticker locator.

Reply:
maybe

Reply:
Not saying right now, but in the future.